lms/djangoapps/grades/rest_api/v1/tests/test_views.py

```diff
@@ -257,3 +260,293 @@ def test_staff_can_get_all_grades(self):
+
+    def test_staff_can_get_all_grades(self):
+        self.client.login(username=self.global_staff.username, password=self.password)
+        resp = self.client.get(self.get_url(urlencode({'course_id': self.course_key})))
+
+        # This should have permission to access this API endpoint.
+        assert resp.status_code == status.HTTP_200_OK
+        expected_data = OrderedDict([('next', None),
+                                     ('previous', None),
+                                     ('results',
+                                      [{'course_id': str(self.course_key),
+                                        'current_grade': 0,
+                                        'passed': False,
+                                        'section_breakdown': self.section_breakdown,
+                                        'username': 'student'},
+                                          {'course_id': str(self.course_key),
+                                           'current_grade': 0,
+                                           'passed': False,
+                                           'section_breakdown': self.section_breakdown,
+                                           'username': 'other_student'},
+                                          {'course_id': str(self.course_key),
+                                           'current_grade': 0,
+                                           'passed': False,
+                                           'section_breakdown': self.section_breakdown,
+                                           'username': 'program_student'},
+                                          {'course_id': str(self.course_key),
+                                           'current_grade': 0,
+                                           'passed': False,
+                                           'section_breakdown': self.section_breakdown,
+                                           'username': 'program_masters_student'}])])
+        assert expected_data == resp.data
+
+    def test_staff_can_get_all_grades_for_user(self):
+        self.client.login(username=self.global_staff.username, password=self.password)
+        resp = self.client.get(self.get_url(urlencode({'course_id': self.course_key,
+                                                       'username': 'student'})))
+
+        # this should have permission to access this API endpoint
+        assert resp.status_code == status.HTTP_200_OK
+        expected_data = OrderedDict([('next', None),
+                                     ('previous', None),
+                                     ('results',
+                                      [{'course_id': str(self.course_key),
+                                        'current_grade': 0,
+                                        'passed': False,
+                                        'section_breakdown': self.section_breakdown,
+                                        'username': 'student'}])])
```

Comment:
```suggestion
        expected_data = OrderedDict(
            [
                ('next', None),
                ('previous', None),
                (
                    'results',
                    [
                        {
                            'course_id': str(self.course_key),
                            'current_grade': 0,
                            'passed': False,
                            'section_breakdown': self.section_breakdown,
                            'username': 'student',
                        }
                    ],
                ),
            ]
        )
```

Nit: suggested a more readable formatting.